languages/csharp/exercises/concept/chars/.docs/instructions.md

```diff
@@ -0,0 +1,43 @@
+In this exercise you will implement a partial set of utility routines to help a developer
+clean up identifier names.
+
+In the 4 tasks you will gradually build up the routine `Clean` A valid identifier comprises
```

Comment:
Story-wise, having empty identifiers is the only thing left that might not make much sense (how would one use an empty identifier?). Was there a specific reason that you wanted to allow for empty identifiers?

Reply:
I will raise a subsequent PR to tweak the wording.  I want to avoid checks for empty output strings as that will increase complexity.  I currently have no feel for the sort of problems exercises pose to presenters but can only imagine that simpler is better.

Reply:
Okay, makes sense.